src/templates/ar/2019/base.html

```diff
@@ -97,10 +146,45 @@
         var menuIcon = document.querySelector('.menu-icon');
         var menuIconInitialSrc = document.querySelector('.menu-icon').src;
         var menuIconSrcClose = document.querySelector('.menu-icon').dataset.srcClose;
+        var menuBtn = document.querySelector('.menu-btn');
+        var menuNav = document.querySelector('#Menu');
+        var menu = document.querySelector('#Menu');
+
+        function triggerMenu() {
+          menuBtn.click();
+        }
 
-        document.querySelector('.menu-btn').addEventListener('click', function() {
+        menuBtn.addEventListener('click', function() {
           var menuOpen = document.body.classList.toggle('menu-open');
           menuIcon.src = menuOpen ? menuIconSrcClose : menuIconInitialSrc;
+          menuBtn.setAttribute('aria-expanded',menuOpen);
```

Comment:
```suggestion
          menuBtn.setAttribute('aria-expanded', menuOpen);
```

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/666/commits/1d5172131674bfb0a4e5f21ed6431d4a09e56eef